.github/workflows/test.yml

```diff
@@ -0,0 +1,71 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+
+jobs:
+  tests:
+    name: Tests
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:11
+        env:
+          POSTGRES_USER: chatruby
+          POSTGRES_DB: chatruby_test
+          POSTGRES_PASSWORD: 'chatruby'
```

Comment:
Please, try to protect secrets values?

Reply:
I don't think this is a big deal since I'm creating a service on the fly for the Postgres database which is a docker instance that is going to be destroyed at the end of the workflow. And since this docker instance (postgres) is not a self hosted runner, we just need to assign needed values for the required env variables in order to be able to use it in the Tests job (line 56) where we need those env varriables.
That's what I think, and actually all of the examples I've seen address this situation the same way. Maybe I'm wrong, if that the case please share with us some examples or resources on how to implement them in the way you think is the correct one and I'll implement that.

Reply:
Thanks!